### PR TITLE
Skoda: add fw for Superb 2020 

### PR DESCRIPTION
**Pull request by Mejty38**
dongle_id: b00fb885011f0384
route_name: b00fb885011f0384|2023-05-10--08-51-48

### CODE REVIEW

File: selfdrive/car/volkswagen/values.py [hunk 1]
```diff
@@ -1257,14 +1258,17 @@ def init_make(self, CP: car.CarParams):
...
       b'\xf1\x870D9300043F \xf1\x895202',
+      b'\xf1\x870GC300019G \xf1\x892803',
     ],
     (Ecu.srs, 0x715, None): [
       b'\xf1\x875Q0959655AE\xf1\x890130\xf1\x82\x12111200111121001121110012211292221111',
       b'\xf1\x875Q0959655AE\xf1\x890130\xf1\x82\022111200111121001121118112231292221111',
       b'\xf1\x875Q0959655AK\xf1\x890130\xf1\x82\022111200111121001121110012211292221111',
       b'\xf1\x875Q0959655BH\xf1\x890336\xf1\x82\02331310031313100313131013141319331413100',
       b'\xf1\x875Q0959655CA\xf1\x890403\xf1\x82\x1331310031313100313151013141319331423100',
-      b'\xf1\x875Q0959655CH\xf1\x890421\xf1\x82\x1333310031313100313152025350539331463100',
```
Comment (incognitojam) [inline selfdrive/car/volkswagen/values.py:1267]: why did you remove this fw version?

Comment (Mejty38) [reply selfdrive/car/volkswagen/values.py:1267]: to be honest i dont even remember removing it. What can I do with it?
